tools/EventClients/Clients/PS3 BD Remote/ps3_remote.py

```diff
@@ -95,7 +95,7 @@ def get_remote_address(remote, target_name = "BD Remote Control"):
                 remote.connect((target_address,19))
                 target_connected = True
                 print "Remote Paired.\a"
-                xbmc.send_notification("Pairing Successfull",
```

Comment:
```
xbmc.send_notification("Action Required!",
xbmc.send_notification("Error", "Unable to find devices.", bticon)
xbmc.send_notification("Found Device",
xbmc.send_notification("Pairing Successful",
xbmc.send_notification("Pairing Failed",
xbmc.send_notification("Error", "No remotes were found.", bticon)
self.xbmc.send_notification("PS3 Blu-Ray Remote", "New Connection", None)
```

I'd rather not make non-spelling changes in these series. Case changes like what you're suggesting should be done w/ an awareness of context and some user testing...